cmd/jag/commands/util.go

```diff
@@ -90,8 +99,15 @@ func (s *SDK) validate() error {
 		} else if stat.IsDir() {
 			return fmt.Errorf("invalid Toit SDK, '%s' was a directory", p)
 		}
+	}
 
+	if !skipSDKVersionCheck {
+		info := GetInfo(ctx)
+		if info.SDKVersion != s.Version() {
+			return fmt.Errorf("SDK Version mismatch. Jaguar SDK version '%s', loaded SDK version '%s'", info.SDKVersion, s.Version())
```

Comment:
Jaguar SDK Version -> Required SDK version is

loaded SDK version -> downloaded SDK version is

Maybe point folks towards running 'jag setup' again?